### ISSUE TITLE
For computed cluster markers, no link outs to gene resources

### ISSUE THREAD

**Issue by matthewspeir** [maximilianh/cellBrowser#70]
It would be nice if the Marker Genes pop-up included links to external resources (e.g. UCSC GB, GeneCards, HGNC) when the markers are computed.

I think you're using GENCODE to define genes, so it seems like we could incorporate GENCODE metadata files (e.g. https://www.gencodegenes.org/human/release_29.html) to make link outs to a few different resources as well. I guess this assumes a more human-focused Cell Browser, but maybe there could be an option to not include these links.

**Comment by maximilianh** [comment]
I don't want to add too many links as people are quickly getting tired of
seeing the same information everywhere. Which links do you think would be
useful to have?

**Comment by matthewspeir** [comment]
I think GeneCards would be best as it contains a lot of information. Basically, I just want a link to a place with info about gene function.

Thanks!

**Comment by maximilianh** [comment]
Let me know how cbMarkerAnnotate works for you. It may be what you wanted. In that case, we can close this ticket.

What this brings up is that cbMarkerAnnotate has no docs right now. Where do you think should I doc it?

**Comment by matthewspeir** [comment]
I'll play with cbMarkerAnnotate a little bit and let you know how it goes. 

And let me think about cbMarkerAnnotate documentation a little bit more. I need to go through the current documentation first and then think about how this new documentation might fit into this.

**Comment by matthewspeir** [comment]
cbMarkerAnnotate is great. Maybe you can add 'GeneCards' to cbMarkerAnnotate? I feel that would be satisfactory. 

I do still think it would nice to add GeneCards to the output 'markers.tsv' by default with the ability to add more link outs with cbMarkerAnnotate, but I'm sure that's more work, haha.

**Comment by maximilianh** [comment]
Good idea. What do you think: How would that link look like in the table ?

**Comment by matthewspeir** [comment]
Hmm, maybe the 'Gene Lists' column should be renamed to something like 'Phenotype and Disease' or something as the sites in that column (OMIM, HPO, and COSMIC) are phenotype/disease databases? 

The 'Gene Lists' column could then be repurposed to contain links out to things like GeneCards? Or maybe it could be in a column named 'Gene Details'?

**Comment by maximilianh** [comment]
Ah nice! Yes, it should be called Phenotype & Disease. Yay browser QA!

The problem is that the normal markers.tsv doesn't have a gene list
column... I could add one... I could add a Gene Details column, always. But
details is also very vague... How about a column "Gene Cards" and a little
"GC" link? Or some Icon. e.g. genecard's logo, the little double helix (how
unusual! :-)

**Comment by matthewspeir** [comment]
I think just a Gene Cards column with the GeneCards logo being the link (or maybe even  'GC {LOGO}') would be good.
